delira/models/model_utils.py

```diff
@@ -0,0 +1,26 @@
+import contextlib
+
+try:
+    # use apex loss scaling if possible
+    # (and enabled, this is done internally by apex)
+    from apex import amp
+except ImportError:
+    # use no loss scaling with same API if apex is unavailable
+    amp = None
+
+
+@contextlib.contextmanager
+def scale_loss(loss,
```

Comment:
Maybe add `**kwargs` if the apex team decides to add some additional arguments in the future.

Reply:
Added by https://github.com/justusschock/delira/pull/97/commits/ea687b3411b5cd61ea49c455da263c40fd1ca3d8